### ISSUE TITLE
Explanation about procotols://

### ISSUE THREAD

**Issue by djalmaaraujo** [includable/react-native-map-link#12]
Hey @tschoffelen I am trying to understand why this is the opposite:

```js
 'apple-maps': isIOS ? 'http://maps.apple.com/' : 'applemaps://',
  'google-maps': isIOS ? 'comgooglemaps://' : 'https://maps.google.com/',
```

Why in IOS we dont use applemaps:// ?

Reference: https://github.com/includable/react-native-map-link/commit/2daad68ca53578c3a6bf57827fc5ecbf97df0b8b

**Comment by tschoffelen** [comment]
The `applemaps://` URI scheme does not exist as far as I know. This is more or less a hacky to make sure Apple Maps does not show up as an option on Android. `maps.apple.com` is the only officially supported way to open the Apple Maps app.

**Comment by djalmaaraujo** [comment]
@tschoffelen Got it, thanks. I am planning to implement `sourceAddress` and `address` to be used instead of latitude/longitude. This is something useful when you don't want to show numbers in the from/to of the apps, you can display addresses, it's more beautiful for the end user.

I could not find a way to pass the latitude/longitude + the name of the place, and the app would show in the Destination field: "My Location Name", but under the hood, use the lat/long.

I tried with `?q=` + saddr/daddr, but no success.

**Comment by tschoffelen** [comment]
Solved by #15 I think, new NPM release is coming in a few mins.
